Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -1565,17 +1565,21 @@ public static InputDevice GetDevice(string nameOrLayout)
         public static TDevice GetDevice<TDevice>()
             where TDevice : InputDevice
         {
-            TDevice result = null;
+            return (TDevice)GetDevice(typeof(TDevice));
+        }
+
+        public static InputDevice GetDevice(Type type)
```

Comment:
Docs.

Reply:
The new `RemoveDevice` behavior is probably worth documenting as well. Something like

```
/// Removing a device that is the current device of its type (see <see cref="InputDevice.MakeCurrent"/>)
/// automatically makes the next device of a compatible type current. For example, removing a <see cref="Mouse"/>
/// may thus make a <see cref="Touchscreen"/> the current <see cref="Pointer.current"/>.
```

Reply:
Hmm, actually just realized that is not what the behavior does. Probably what it should do but not sure we really need to bother.

Reply:
fixed